src/pages/VerdictPage.tsx

```diff
@@ -74,14 +73,20 @@ export default function VerdictPage() {
 
   const verdictProperties = Object.keys(emptyVerdict)
   const [currentVerdict, setCurrentVerdict] = useState<Verdict>(emptyVerdict)
-  const [testCases, setTestCases] = useState<number[]>()
+  const [testCases, setTestCases] = useState<number[]>([0])
+
+  const [code, setCode] = useState<string>('')
+  const [problemName, setProblemName] = useState<string>('')
+  const [verdictNum, setVerdictNum] = useState<number>(0)
+  const [fileType, setFileType] = useState<string>('txt')
 
   const problemsObject = useProblemsObject()
 
   const params = useParams()
   const submissionId: string = params.submissionId as string
 
   useEffect(() => {
+    let intervalId: NodeJS.Timeout
```

Comment:
It doesn't look like this variable is being initialized anywhere. 

It might have been fine before. Since @sd0623 mentioned I guess it was hoisting.